src/connector/connection_info.rs

```diff
@@ -243,6 +277,14 @@ impl SqlFamily {
             _ => false,
         }
     }
+
+    #[cfg(feature = "mssql")]
+    pub fn is_mssql(&self) -> bool {
+        match self {
+            SqlFamily::Mssql => true,
+            _ => false,
+        }
```

Comment:
I've started to use `matches!(self, SqlFamily::Mssql)` for these, but the match is definitely readable too.